packages/core/src/routes/swagger/utils/documents.ts

```diff
@@ -207,7 +213,7 @@ export const buildUserApiBaseDocument = (
 });
 
 export const getSupplementDocuments = async (
-  directory = 'routes',
+  directory = 'build',
```

Comment:
might need some more context for this

Reply:
Based on our previous offline discussion, the code related to the SAML app will be placed in a separate folder. This folder (`/core/src/saml-applications`) will be at the same level as the `/core/src/routes` path. When building swagger.json or other openapi.json files, we will copy all openapi.json files to the `core/build` path and then integrate them uniformly. Since the `/core/src/saml-applications` folder is not under `/core/src/routes` and is not within the scope of the original method, we need to elevate the directory to the build path to include the openapi.json files related to the SAML app. The previous update to the `copy:apidocs` script was for the same reason.

Reply:
Will add comments later.